module/config.mjs

```diff
@@ -405,6 +459,70 @@ ED4E.attributePointsCost = [0, 1, 2, 3, 5, 7, 9, 12, 15];
 ED4E.attributePointsCost[-1] = -1;
 ED4E.attributePointsCost[-2] = -2;
 
+
```

Comment:
There's a lot of whitespace... Please remove :)

Reply:
done